tests/end2end/playwright/print.spec.js

```diff
@@ -337,6 +337,127 @@ test.describe('Print', () => {
     });
 });
 
+test.describe('Print opacities', () => {
+    test.beforeEach(async ({ page }) => {
+        const url = '/index.php/view/map/?repository=testsrepository&project=group_as_layer_opacity';
+        await gotoMap(url, page);
```

Comment:
This `gotoMap` is deprecated, as "new" tests must use `POM` https://playwright.dev/docs/pom. Look in "playwright/pages"
You can add some locators about the "print" box, such as the scale, the buttons...

Reply:
Please also add the tag `@readonly` to your new test.